presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java

```diff
@@ -108,6 +108,7 @@
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.hive.metastore.TableType;
+import org.iq80.leveldb.table.TableBuilder;
```

Comment:
This was accidentally imported and is causing the build to fail. Please remove it.